src/plotloader/PlotLoader.js

```diff
@@ -30,7 +31,8 @@ PlotLoader.propTypes = {
   highchartsConfig: PropTypes.object,
   chartClassName: PropTypes.string,
   resourcesUrl: PropTypes.string,
-  children: PropTypes.object
+  children: PropTypes.object,
```

Comment:
Is the `children` prop needed now that we don't use the separate gradient component?